src/CMakeLists.txt

```diff
@@ -307,6 +316,8 @@ include_directories(${CMAKE_BINARY_DIR}/include)
 # the linker flags necessary on other platforms.
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(LEANC_STATIC_LINKER_FLAGS "-lleancpp -lInit -lStd -lLean")
+elseif(${CMAKE_SYSTEM_NAME} MATCHES "Emscripten")
+  set(LEANC_STATIC_LINKER_FLAGS "-lleancpp -lInit -lStd -lLean -lnodefs.js")
```

Comment:
I read this as "no defs" way too long...

Reply:
Node.js filesystem :)